tests/integration/llm/client.py

```diff
@@ -76,7 +76,19 @@
     "t5-small": {
         "batch_size": [1, 2],
         "max_memory_per_gpu": 2
-    }
+    },
+    "gpt2-xl": {
+        "batch_size": [1, 2],
+        "max_memory_per_gpu": 7.0
+    },
+    "facebook/opt-13b": {
+        "batch_size": [1, 2],
+        "max_memory_per_gpu": 15.0
```

Comment:
same in here. the model is saved in fp16, not making sense to test in fp32

Reply:
updated.